packages/server/src/views/index/user.mustache

```diff
@@ -37,6 +37,7 @@
 
 		<div class="control block">
 			<input type="submit" name="post_button" class="button is-primary" value="{{buttonTitle}}" />
+            <input type="submit" name="send_account_confirmation_email" class="button is-link" value="Send account confirmation email" />
```

Comment:
As I asked in the dev-channel, do I need to add some condition to rather show or don't show this button?